src/main/java/me/John/BossBanner/Main.java

```diff
@@ -85,7 +85,7 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
 			if (args[0].equalsIgnoreCase("TextsReload")) {
 				if (sender instanceof Player) {//player command
 					Player player = (Player) sender;
-					if (player.hasPermission("bossbar.admin")) {//has permission
+					if (player.hasPermission("admin")) {//has permission
```

Comment:
permission strings should include the plugin name: bossbanner.admin
Also permissions should be declared in plugin.yml file
https://bukkit.gamepedia.com/Plugin_YAML

Reply:
Permissions are added as well as renamed.